app/assets/javascripts/components/favorites/favorites.js.coffee

```diff
@@ -0,0 +1,48 @@
+{div, span, br} = React.DOM
+
+window.FavoritesListClass = React.createClass
+
+  displayName: "FavoritesListClass"
+
+  render: ->
+
+    #
+    # Configuration passed to SMaterialIcon
+    #
+    configuration = {
+        enableFavorites:    true,
+        favoriteClassMap:   {
+            true:   "stem-finder-result-favorite-active",
+            false:  "stem-finder-result-favorite"
+        },
+        width:              "300px",
+        height:             "250px"
+    }
+
+    (div { style: { display: "flex", \
+                    flexFlow: "row wrap"} }, 
+
+      for item in @props.items
+        (div {  style: {    width: "300px",     \
+                            flex: "1 1 45%",    \
+                            display: "table" }, \
```

Comment:
@dougmartin since we reverted the favorites page to use the legacy search results view (pending integration with your changes) this code isn't actually being used. I will remove this file.